tests/v8_intrinsic/intrinsic_call.js

```diff
@@ -0,0 +1,5 @@
+function doSmth()     {
+            %DebugPrint
+        (
+                foo )
+  }
```

Comment:
Could you expand the test coverage to have other places where v8 intrinsics are used? One test seems light.

Reply:
Added few more, although it's not very easy to be creative with something like this :)